lib/ExpressionUtils/ExpressionUtilsImpl.py

```diff
@@ -139,12 +139,14 @@ def _get_ws_info(self, obj_ref):
     def _get_genome_ref(self, assembly_or_genome_ref, params):
 
         obj_type = self._get_ws_info(assembly_or_genome_ref)[2]
```

Comment:
Making a workspace call to get the ws_info can be delayed until line 145.

Reply:
@ugswork Done